src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -1077,6 +1077,14 @@ private void ScanInterpolatedStringLiteralHoleBalancedText(InterpolatedStringKin
                                 if (_lexer.TryScanAtStringToken(ref discarded))
                                     continue;
 
+                                if (_lexer.TextWindow.PeekChar(1) == '*')
+                                {
+                                    // Razor comment. Handle for error recovery purposes. The parser will come along later to retokenize the inside of the string
+                                    // and report a proper error.
+                                    _lexer.ScanMultiLineComment(isTerminated: out _, '@');
```

Comment:
same nit.

Reply:
love this.  just asking for tests for this (since i haven't looked at the tests yet).